programs/zstd.1.md

```diff
@@ -254,6 +254,21 @@ Compression of small files similar to the sample set will be greatly improved.
 
     `zstd --train-cover=k=50,split=60 FILEs`
 
+* `--train-fastcover[=k#,d=#,f=#,steps=#,split=#,accel=#]`:
+    Same as cover but with extra parameters f and accel and different default value of split
```

Comment:
`finalize` and `skip` are out of date. Explain what `f` is.

Reply:
Can you please explain what you mean by "finalize and skip are out of date"?

Reply:
A few lines down you mention `finalize` and `skip`, but they don't exist anymore.